### DISCUSSION TITLE
Suggestion: Portrait fixed and animation at the same time for all team members.

### DISCUSSION THREAD

**Discussion by jefinhoyagami** [ikemen-engine/Ikemen-GO#887] [Feature Request]
I would suggest the addition of fixed portrait and animation for each team member mainly in Team mode, it is possible to use this feature but only one member of the team displays the extra portrait, this in the select and mainly in VS mode. The idea is that we have options to mess with those portraits for each member as well as scale position and everything. Example of how the code would be in System.def.

In this you could use the character's animation 0 and of course all members at the same time.
    p1.member1.face.anim = -1
    p1.member1.face.spr = 9000, 1
    p1.member1.face.done.anim = -1
    p1.member1.face.done.spr = 9000, 1
    p1.member1.face.offset = 0, 0
    p1.member1.face.scale = 1.0, 1.0
    p1.member1.face.slide.speed = 0, 0
    p1.member1.face.slide.dist = 0, 0

Already in this you could use a fixed portrait 9000,1 would be a big portrait.

    p1.member1.face2.anim = -1
    p1.member1.face2.spr = 9000, 1
    p1.member1.face2.done.anim = -1
    p1.member1.face2.done.spr = 9000, 1
    p1.member1.face2.offset = 0, 0
    p1.member1.face2.scale = 1.0, 1.0
    p1.member1.face2.slide.speed = 0, 0
    p1.member1.face2.slide.dist = 0, 0

The two would work together as in some kof games.

https://1.bp.blogspot.com/-_YlyrqccxcE/Xw58_zsQYAI/AAAAAAAAv1U/UPA5vxnLIyMW2jVfR8057SR9edkby4_lwCLcBGAsYHQ/s640/mrkarate.png

https://www.fightersgeneration.com/nz2/game/kof2002um-matchup-screen.jpg

Thank you very much for your attention.

**Comment by UltimateM007** [comment]
We really need this. (y)

**Comment by ZeroSombrio** [comment]
**Yeah! That'll be nice for my project too!**

**Comment by anakira** [comment]
Esse é  um recurso fundamental para grandes projetos , essencial para grandes criações.

**Comment by Splashslash** [comment]
Will be a good improvement!

**Comment by gamer981-png** [comment]
![Screenshot (37)](https://github.com/user-attachments/assets/45c89b2e-1a7b-492f-94bc-0c49faa25d96)
![Screenshot (38)](https://github.com/user-attachments/assets/6bc74dec-bf3c-4624-bd0d-310634a1563b)
can someone help with this,cursor only show corners when mugen screenpack  is converted in ikemen go
the issue is I think the window = value but cant find where to place it so I can stretch the view >>>????
THNX
